### PR TITLE
Add kzg to `common.customCrypto`

### PR DESCRIPTION
**Pull request by acolytec3**
This adds `kzg` as an optional property on `common.customCrypto` and updates usage across the monorepo.

It also fixes a gap in our `common.fromGethGenesis` constructor to pass in the custom crypto object from common

### CODE REVIEW

File: packages/common/src/common.ts [hunk 1]
```diff
@@ -1086,6 +1088,20 @@ export class Common {
...
 
+  /**
+   * Initialize the KZG library
+   * @param kzg - a KZG implementation that implements the @ethereumjs/util KZG interface
+   * @param pathToTrustedSetup string representing the path to the trusted setup necessary for
+   * instantiating the KZG object - defaults to the official one shipped with EthereumJS
+   */
+  initializeKZG(kzg: Kzg, pathToTrustedSetup?: string) {
+    try {
+      initKZG(kzg, pathToTrustedSetup ?? __dirname + '/../../client/src/trustedSetups/official.txt')
```
Comment (holgerd77) [inline packages/common/src/common.ts:1099]: This is for sure not working just realizing. 🙂 😂

Comment (acolytec3) [reply packages/common/src/common.ts:1099]: Hah, true.  Going to revert this one.

**Comment by holgerd77** [comment]
Updated this via UI

**Review by holgerd77 — APPROVED** [review]
LGTM

**Pull request merged**